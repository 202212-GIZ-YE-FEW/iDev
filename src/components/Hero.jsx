export default function Hero() {
    return <div></div>;
}
