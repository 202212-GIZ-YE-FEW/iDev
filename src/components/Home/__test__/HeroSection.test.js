import renderer from "react-test-renderer";

import Hero from "../HeroSection";

it("renders correctly", () => {
    const tree = renderer.create(<Hero />).toJSON();
    expect(tree).toMatchSnapshot();
});
