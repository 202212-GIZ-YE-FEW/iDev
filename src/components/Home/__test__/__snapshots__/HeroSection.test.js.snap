// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header
  className="min-w-[96vw] bg-cover"
  style={
    Object {
      "backgroundImage": "url(/header.png)",
      "minHeight": "calc(100vh - 5rem)",
    }
  }
>
  <div
    className="container flex flex-col h-full justify-center lg:flex-row gap-y-10 lg:justify-between items-center"
  >
    <div
      className="flex flex-col justify-center text-center lg:text-start"
    >
      <h1
        className="text-xl sm:text-3xl md:text-4xl xl:text-5xl uppercase"
      >
        weAreHereTo
      </h1>
      <p
        className="text-3xl sm:text-5xl md:text-9xl rtl:lg:text-9xl mb-3 lg:mb-0 uppercase"
      >
        help
      </p>
      <a
        className="w-fit xl:w-full rtl:w-full"
        href="#"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <button
          className="font-normal whitespace-nowrap cursor-pointer  capitalize text-lg md:text-xl lg:text-2xl rounded-md px-[25px] md:px-[35px] lg:px-[59px] py-[8px] font-medium bg-cyan text-black "
        >
          bookAppointment
        </button>
      </a>
    </div>
    <div
      className="basis-1/2"
    >
      <img
        alt=""
        className="object-fit w-full lg:min-w-[29rem] lg:max-w-[36rem] xl:min-w-[40rem]"
        data-nimg="1"
        decoding="async"
        height={200}
        loading="lazy"
        onError={[Function]}
        onLoad={[Function]}
        src="/_next/image?url=%2Fhero.png&w=1920&q=75"
        srcSet="/_next/image?url=%2Fhero.png&w=1080&q=75 1x, /_next/image?url=%2Fhero.png&w=1920&q=75 2x"
        style={
          Object {
            "color": "transparent",
          }
        }
        width={900}
      />
    </div>
  </div>
</header>
`;
